### PR TITLE
creating alb health alert policies

### PR DESCRIPTION
**Pull request by mrajess**
This is the first of two PRs to address issue #153. This is my first PR against this repo. I tried to follow what appeared to be the standards. Please let me know if I missed anything or if you see any errors.

### CODE REVIEW

**Review by jfaurskov — COMMENTED** [review]
Provided feedback in above comment

**Review by jfaurskov — CHANGES_REQUESTED** [review]
Please see previous comments for changes requested. :-)

File: src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_datapathavailability_alert.bicep [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+]
+
+@allowed([
+    '0'
+    '1'
+    '2'
+    '3'
+    '4'
+])
+param parAlertSeverity string = '0'
```
Comment (arjenhuitema) [inline src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_datapathavailability_alert.bicep:18]:
```suggestion
param parAlertSeverity string = '1'
```

@jfaurskov if I remember correctly, this type of alert we would classify as a 1, correct?

File: src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_healthprobestatus_alert.bicep [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+    'deployIfNotExists'
+    'disabled'
+])
+param parPolicyEffect string = 'deployIfNotExists'
+
+param parAutoMitigate string = 'true'
+
+param parAlertState string = 'true'
+
+param parThreshold string = '100'
```
Comment (arjenhuitema) [inline src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_healthprobestatus_alert.bicep:51]:
```suggestion
param parThreshold string = '90'
```

File: src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_usedsnatports_alert.bicep [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+]
+
+@allowed([
+    '0'
+    '1'
+    '2'
+    '3'
+    '4'
+])
+param parAlertSeverity string = '0'
```
Comment (arjenhuitema) [inline src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_usedsnatports_alert.bicep:18]:
```suggestion
param parAlertSeverity string = '1'
```

**Review by arjenhuitema — CHANGES_REQUESTED** [review]
Great job! I´ve added a few suggestions.

File: src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_healthprobestatus_alert.bicep [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+]
+
+@allowed([
+    '0'
+    '1'
+    '2'
+    '3'
+    '4'
+])
+param parAlertSeverity string = '0'
```
[the remark this answers is not in the document]
Comment (jfaurskov) [reply src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_healthprobestatus_alert.bicep:18]: @arjenhuitema probably something we'll want to discuss tmr.

File: src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_usedsnatports_alert.bicep (hunk 3 above)
Comment (jfaurskov) [reply src/resources/Microsoft.Authorization/policyDefinitions/deploy-alb_usedsnatports_alert.bicep:18]: As discussed at the meeting on Friday, we'll go with this (0) for now, no need to immediately change this.

**Comment by arjenhuitema** [comment]
Issue#153 will be addressed in PR#212

**Pull request closed without merging**